mitosheet/src/plugin.tsx

```diff
@@ -8,7 +8,7 @@
 import { IJupyterWidgetRegistry } from '@jupyter-widgets/base';
 import { INotebookTracker, NotebookActions } from '@jupyterlab/notebook';
 import { Application, IPlugin } from 'application';
-import { Widget } from 'widgets';
+import { Widget } from "../node_modules/@lumino/widgets/types/widget"
```

Comment:
This is def wrong!

Reply:
These paths don't make sense when things are built. There are no node modules distributed to the user!